common/convert/convert.go

```diff
@@ -59,11 +59,10 @@ func TimeFromUnixTimestampFloat(raw interface{}) (time.Time, error) {
 	return time.UnixMilli(int64(ts)), nil
 }
 
-// TimeFromUnixTimestampDecimal converts a unix timestamp in decimal form to
-// a time.Time
+// TimeFromUnixTimestampDecimal converts a unix timestamp in decimal form to a time.Time
 func TimeFromUnixTimestampDecimal(input float64) time.Time {
 	i, f := math.Modf(input)
-	return time.Unix(int64(i), int64(f*(1e9)))
+	return time.Unix(int64(i), int64(f*(1e9))).UTC()
```

Comment:
Suggestion: just noticed `i` might be less than zero if input is jank. So maybe a check for negative numbers and return time.Time{}?

Reply:
I don't think this is wrong.
If you meant negative when you said _jank_ then I think the behaviour is [correct](https://go.dev/play/p/fcRojpcKpk6)

I couldn't find an outcome I was unhappy with.
Could you provide an example?

Reply:
The time is before 1970 so if you Unix() that it would be `-31536000`, I suggested time.Time{} so you could do an IsZero() call on it 🤷 . Up to you if you want to change it, if it is fine in your eyes just resolve.

Reply:
I think that if you give a negative number for a Unix timestamp, at *best* you could expect to get that date before Unix epoch. Returning an empty time would be wrong IMO.